src/Makefile.legacy

```diff
@@ -573,7 +573,7 @@ linux-x86-64-32-any:
 		LDFLAGS="$(LDFLAGS) -m32 -lcrypt -ldl" \
 		AESNI_ARCH=86 YASM_FORMAT="elf32"
 	@echo "Failing after this point just means some helper tools did not build:"
-	$(MAKE_ORIG) $(PROJ_PCAP) CFLAGS="$(CFLAGS) -m32"
+	$(MAKE_ORIG) $(PROJ_PCAP) CFLAGS="$(CFLAGS) -m32" LDFLAGS="$(LDFLAGS) -m32"
```

Comment:
Isn't this a hint that #5240 '-m64' is needed, at least in the meantime?

Reply:
No, needing to use `-m32` to build/link 32-bit code on a typical 64-bit system is a hint that such system actually _doesn't_ need explicit `-m64` to build/link 64-bit code.